File: Sharebook/Client/src/app/core/Core/DTO/EventInitiatedViaDTO.ts
import {IEventInitiatedViaDTO} from "../../Public/Models/Interfaces/DTO/IEventInitiatedViaDTO";

export class EventInitiatedViaDTO implements IEventInitiatedViaDTO {
  // [JsonProperty("tweet_id")]
  public tweetId: number;

  // [JsonProperty("welcome_message_id")]
  public welcomeMessageId?: number;
}

File: Sharebook/Server/src/Sharebook.Common/DTO/EventInitiatedViaDTO.cs
namespace Sharebook.Common.DTO
{
    using Newtonsoft.Json;
    using Sharebook.Common.Public.Models.Interfaces.DTO;

    public class EventInitiatedViaDTO : IEventInitiatedViaDTO
    {
        [JsonProperty("tweet_id")]
        public long TweetId { get; set; }

        [JsonProperty("welcome_message_id")]
        public long? WelcomeMessageId { get; set; }
    }
}
